.gitignore

```diff
@@ -47,6 +47,9 @@ src-shared/
 # output for docs-platform
 json/
 
+# output of scripts
+scripts/frontmatter_tagging/tmp/*
```

Comment:
I put all the files into a tmp folder instead of having them replace `_cci2` or `_cci2_ja2` to allow for spot checks before committing the script.